irma/ftp/sftp.py

```diff
@@ -53,7 +53,12 @@ def _connect(self):
             self._conn.window_size = pow(2, 27)
             self._conn.packetizer.REKEY_BYTES = pow(2, 32)
             self._conn.packetizer.REKEY_PACKETS = pow(2, 32)
-            self._conn.connect(username=self._user, password=self._passwd)
+            if self._auth == 'key':
+                pkey = RSAKey.from_private_key_file('/home/irma/.ssh/id_rsa')
```

Comment:
seems strange to login as frontend or probe user with a key in irma's home directory. Means also that all accounts are available with a single key.